releases/latest/full/Dockerfile.ubi.openjdk21-new

```diff
@@ -0,0 +1,32 @@
+# TODO 
+#  Check that the parent image is correct
+
+ARG PARENT_IMAGE=openliberty/open-liberty:24.0.0.1-kernel-slim-java21-openj9-ubi-minimal
```

Comment:
Use the following base image:
`icr.io/appcafe/open-liberty:kernel-slim-java21-openj9-ubi-minimal`

Base image should be from icr.io. The source in `latest` should not include the version in the tag (i.e. 24.0.0.1)